python/cudf/cudf/core/dataframe.py

```diff
@@ -6731,6 +6731,14 @@ def append(
 
         return cudf.concat(to_concat, ignore_index=ignore_index, sort=sort)
 
+    def pivot(self, index, columns, values=None):
+        return cudf.core.reshape.pivot(
+            self, index=index, columns=columns, values=values
+        )
+
+    def unstack(self, level=-1):
```

Comment:
I understand the need to match the Pandas API so that things like Dask don't break. But what if downstream code is relying on the behaviour of the `fill_value`? Wouldn't it be better that they got a `TypeError` here, rather than us silently ignoring the argument?

Reply:
We shouldn't ignore the argument, we should throw if someone passes a value to `fill_value` that we don't support.

Reply:
👍